vision/module/in_frame_driver.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
Let's delete this file and move this into the if __name__ == '__main__': of in_frame.